AERX-dev/aerx-frontend-near
# ft(earn2gether): added ui

## What this PR adds

The PR adds Earn2Gether feature.

## Feature Description

An authorized user should be able to re-share a post not  on his/her profile.

## How to test this

Open the vercel preview link. To test the feature.
Alternatively, check into the branch `ft-earn2gether` and click share on flow.

## Screenshots

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/54368821/181917151-110a0253-54af-43d1-9717-297dd4624802.png">
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/54368821/181917183-59a4703e-00d8-4f02-a2ef-98d75df31db8.png">
